.github/workflows/codeql.yml

```diff
@@ -25,7 +26,9 @@ on:
       - ready_for_review
     branches: [ "main" ]
     paths-ignore:
-      - '.github/**'
+      - '**.go'
+      - '**.js'
```

Comment:
why we are adding and then at the same time ignoring here?

Reply:
Good catch. We don't want to ignore here, only in the `*-required` flows.